Description: Fix translation's language codes -- also see debian/rules (#336799)
 Using a country part in a translation file is a discouraged practice
 except in very few cases (pt_BR, zh_CN, zh_TW). -- Christian Perrier
Author: Moray Allan <moray@debian.org>
Last-Update: 2012-03-21
--- a/configure.ac
+++ b/configure.ac
@@ -95,7 +95,7 @@
 
 if test x$enable_nls = xyes; then
 
-  LINGUAS="en_GB es_ES fi_FI de cs fr_FR zh_TW"
+  LINGUAS="en es fi de cs fr zh_TW"
 
   AC_CHECK_FUNC(gettext, [HAVEGETTEXT="yes"], 
 	     AC_CHECK_LIB(intl, gettext, [INTLIBS="-lintl" HAVEGETTEXT="yes"],
--- a/po/Makefile.am
+++ b/po/Makefile.am
@@ -3,7 +3,7 @@
 CLEANFILES = $(CATALOGS) matchbox.pot
 
 # keep this sorted
-EXTRA_DIST = en_GB.po es_ES.po fi_FI.po de.po cs.po fr_FR.po zh_TW.po
+EXTRA_DIST = en.po es.po fi.po de.po cs.po fr.po zh_TW.po
 
 POTFILES  = \
 	$(top_builddir)/src/panel_menu.c   \
